.travis.yml

```diff
@@ -25,6 +25,7 @@ before_deploy:
 - >
   if ! [ "$BEFORE_DEPLOY_RUN" ]; then
     export BEFORE_DEPLOY_RUN=1;
+    export ASCIIDOCTOR_CORE_VERSION=v1.5.6;
```

Comment:
I think we need to set `ASCIIDOCTOR_CORE_VERSION` to release Asciidoctor.js against the latest Asciidoctor core release.

As far as I understand, `before_deploy` task is triggered once all the builds (matrix) are successful ? So I don't really know what version of Node and what version of Asciidoctor core will be used... ? Any idea ?